bin/get-video-s3-acls.py

```diff
@@ -0,0 +1,309 @@
+#!/usr/bin/env python3
+# Permissions required:
+#  psql - access to heroku app's postgres
+#  aws s3 - NZSL IAM access
+#  s3:GetObjectAcl permissions or READ_ACP access to the object
+#  https://docs.aws.amazon.com/cli/latest/reference/s3api/get-object-acl.html
+
+
+import os
+import sys
+import subprocess
+import argparse
+import json
+
+# TODO
+# We are using external apps just for the moment.
+# These will be removed for native libraries.
+AWSCLIENT = "/usr/local/bin/aws"
+PGCLIENT = "/usr/bin/psql"
+
+# NZSL: Is there a database url defined in the environment?
+DATABASE_URL = os.getenv("DATABASE_URL", None)
+
+parser = argparse.ArgumentParser(
+    description="You must have a configured AWSCLIENT profile to use this app. See the --awsprofile "
+    "argument."
+)
+# Positional arguments
+if DATABASE_URL:
+    print("DATABASE_URL defined in environment", file=sys.stderr)
+else:
+    parser.add_argument(
+        "dburl",
+        help=f"(REQUIRED) Database url (Overridden by DATABASE_URL environment variable)",
+    )
```

Comment:
issue: I think we should just drop this and rely on the `DATABASE_URL` env variable

(especially given that it will include the password so not something you want to be passing as plaintext in a command)

Reply:
Done.